app/src/main/scala/org/alephium/explorer/api/model/StdInterfaceId.scala

```diff
@@ -25,37 +25,44 @@ import org.alephium.util.Hex
 sealed trait StdInterfaceId {
   def value: String
   def id: String
+  def category: String
 }
 
 object StdInterfaceId {
 
   case object FungibleToken extends StdInterfaceId {
-    val value: String = "fungible"
-    val id: String    = "0001"
+    val value: String    = "fungible"
+    val id: String       = "0001"
+    val category: String = "0001"
   }
 
   case object NFTCollection extends StdInterfaceId {
-    val value: String = "non-fungible-collection"
-    val id: String    = "0002"
+    val value: String    = "non-fungible-collection"
+    val id: String       = "0002"
+    val category: String = "0002"
   }
 
   case object NFTCollectionWithRoyalty extends StdInterfaceId {
-    val value: String = "non-fungible-collection-with-royalty"
-    val id: String    = "000201"
+    val value: String    = "non-fungible-collection-with-royalty"
+    val id: String       = "000201"
+    val category: String = "0002"
   }
 
   case object NFT extends StdInterfaceId {
-    val value: String = "non-fungible"
-    val id: String    = "0003"
+    val value: String    = "non-fungible"
+    val id: String       = "0003"
+    val category: String = "0003"
   }
 
   final case class Unknown(id: String) extends StdInterfaceId {
-    val value: String = s"$id"
+    val value: String    = s"$id"
+    val category: String = ""
```

Comment:
Let's assume that all valid interface id should follow these rules:
1. the category is always the first two bytes
2. the id should have at least two bytes

Reply:
perfect,  so I could just do `id.take(4)` for the category. I will also add a validation that an idea needs at least 2 bytes.